layout/_partial/sidebar-footer_image.ejs

```diff
@@ -0,0 +1,9 @@
+<% if(theme.img.footer_image) { %>
+    <div id="footer-image">
```

Comment:
- I suggest you to put the `div` inside the `for` loop
- Please change `#scheme-Paradox #upyun-logo` by `#scheme-Paradox #footer-image` in `style.css`
- Please test with multiple images to adapt the style